ehsm_kms_service/user_info.js

```diff
@@ -0,0 +1,4 @@
+const user_info = {
+    "202112101919":"202112345678"

```

Comment:
what's this for?

Reply:
This is a temporary database used to obtain appkey matching appid. Later, this content will be migrated to the database